src/main/java/org/gridsuite/shortcircuit/server/entities/FeederResultEntity.java

```diff
@@ -66,4 +71,21 @@ public FeederResultEntity(String connectableId, double current, FortescueResultE
         this.current = current;
         this.fortescueCurrent = fortescueCurrent;
     }
+
+    public boolean match(ResourceFilter filter) {
+        // FeederResultEntity may only be filtered through connectableId
+        if (filter.column().equals(FeederResultEntity.Fields.connectableId) &&
+            filter.dataType() == TEXT) {
+            switch (filter.type()) {
+                case EQUALS: return connectableId.equals(filter.value().toString());
```

Comment:
There is a new switch syntax with arrow for this kind of use case

Reply:
Nice ! Done.